doc/RELEASE_NOTES.md

```diff
@@ -30,6 +30,7 @@ We did not bump the "major" release version number with these changes because we
 
 Note that work on the `v0.71` release started in earnest with the `v0.69.3-agent` release (tagged as `b0.69-bp`).  A number of bug fixes and behaviors from the `v0.71` work have already been back-ported and delivered in the various `v0.69.*` releases since then.  These release notes will highlight only the behavioral changes that have not been back-ported previously.
 
+This release supports RHEL 7.9, RHEL8.6, RHEL 9, CentOS-Stream 8, CentOS-Stream 9 and Fedora 35. For various reasons, it does *NOT* support RHEL 8.x (where x <= 5), RHEL 9.1 or Fedora 36. If you need support for any of these, please talk to us: we will do our best to accommodate you in some way, but there is no guarantee.
```

Comment:
I have hesitated for minutes to even question this, but I'm uncomfortable with dumping these all into the same bucket:

- RHEL 8.4 / 8.5 are problems due to the ansible/ansible-core changes, which aren't well supported in the standard repos ... but you have a workaround by enabling additional support repos. I'm not sure I'd want the release notes to go into detail, but they could be called out as having "known workarounds"?
- RHEL 9.1 is just "immature" and doesn't have all the support repos available yet, a problem likely to be resolved by time though some adjustments on our part might be necessary?
- Fedora 36 and Python3.10 present enormous issues we haven't yet figured out how to deal with; pretty sure at this point we don't even want to suggest we're willing to get drawn into that, making me wary of the "we will do our best to accommodate you" line, which seems more appropriate for the RHEL 8.x/9.1 entries...

Reply:
The weasel words are "For various reasons ..." :-) I wanted the release notes to contain the "what", not the "why" - but if there is some consensus, I'm open to changing the wording.

Reply:
I added short parenthetical explanations of the various reasons.

Reply:
@dbutenhof, can you check the change and see if that resolves the converstaion? I'd like to merge this.

Reply:
NM  - I just made an update to get the latest commits into the list.  This PR will be the first commit *after* the "official" tag point of the release in any case.